recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - python
     - requests >=2.20.0
     - natsort >=4.0.3
-    - {{ pin_compatible('numpy') }}
+    - numpy >= 1.17.0
```

Comment:
I think numpy from host is injecting this now, so this might not be needed explicitly here.

Reply:
@jaimergp Thanks for looking into my PR. I am trying to update version AND build using NumPy 2.0 simultaneously. Is there an easier way to achieve this other than manually editing the entire PR?

Reply:
@jaimergp Problem solved in the other PR. Closing. Thanks!